app/src/main/java/com/linecorp/linesdk/api/LineApiTestClientFactory.java

```diff
@@ -22,4 +24,14 @@ public static LineApiClient createLineApiClient(
                 .apiBaseUri(Uri.parse(BuildConfig.API_SERVER_BASE_URI))
                 .build();
     }
+
+    @NonNull
+    public static OpenChatApiClient createOpenchatApiClient(
+            @NonNull final Context context,
+            @NonNull final String channelId) {
+        return new OpenChatApiClientImpl(
+                context,
+                Uri.parse(BuildConfig.API_SERVER_BASE_URI),
+                new AccessTokenCache(context, channelId));
+    }
```

Comment:
I think we can move `OpenChatApiClient` to `LineApiClient` and expose open chat related API, we can consider `LineApiClient` as a facade for developers who are using LINE SDK. In addition, the `OpenChatApiClient` interface can be changed to implementation directly.

```kotlin
public interface LineApiClient {
    ...

    @NonNull
    LineApiResponse<Boolean> getAgreementStatus();

    @NonNull
    LineApiResponse<Boolean> updateAgreementStatus(@NonNull Boolean agreed);

    @NonNull
    LineApiResponse<OpenChatRoomInfo> createOpenChatRoom(@NonNull OpenChatParameters openChatParameters);

    @NonNull
    LineApiResponse<OpenChatRoomStatus> getOpenChatRoomStatus(@NonNull String roomId);

    @NonNull
    LineApiResponse<MembershipStatus> getMembershipStatus(@NonNull String roomId);

    ...
}
```
```kotlin
public 
class LineApiClientImpl implements LineApiClient {
    ...
    @NonNull OpenChatApiClient openChatApiClient
    ...
    public LineApiClientImpl(
            ...
            @NonNull OpenChatApiClient openChatApiClient) {
        ...
        this.openChatApiClient = openChatApiClient;
        ...
    }

    // override open chat related api
}
```